src/main/java/org/folio/services/validator/InvoiceValidator.java

```diff
@@ -85,7 +92,7 @@ public void validateInvoiceTotals(Invoice invoice) {
     }
   }
 
-  private boolean isAdjustmentIdsNotUnique(List<Adjustment> adjustments) {
+  public static boolean isAdjustmentIdsNotUnique(List<Adjustment> adjustments) {
```

Comment:
The same here, you don't need static here, you can use this validator as in other places using constructor: new InvoiceValidator();

Reply:
done